utility/src/main/java/org/oppia/util/parser/ImageLoaderAnnotation.kt

```diff
@@ -0,0 +1,7 @@
+package org.oppia.util.parser
+
+import javax.inject.Qualifier
+
+/** Corresponds to image loader. */
+@Qualifier
+annotation class ImageLoaderAnnotation
```

Comment:
Do we actually need this annotation? We should just be able to inject ``ImageLoader`` directly. Please remove, or follow up with me on why we need this.

Reply:
If I remove and directly inject ImageLoader its throwing me error 
` error: cannot access SimpleTarget
  class file for com.bumptech.glide.request.target.SimpleTarget not found`

Reply:
This is been resolved.